schema/google/showcase/v1beta1/echo.proto

```diff
@@ -273,6 +283,23 @@ message ErrorWithMultipleDetails {
   repeated google.protobuf.Any details = 1;
 }
 
+// The custom error detail to be included in the error response from the
+// FailEchoWithDetails method.
+message PoetryError {
+  string poem = 1;
+}
+
+// The request message used for the FailEchoWithDetails method.
+message FailEchoWithDetailsRequest {
+  // empty request messages don't work with the CLI: https://github.com/googleapis/gapic-generator-go/issues/1616
+  string unused = 1;
```

Comment:
Not that it's super duper helpful, but maybe just use this field to populate the custom error? To ensure that it get's fully parsed and such? Just an idea